lms/static/sass/theme-overrides.scss

```diff
@@ -1,93 +1,193 @@
+            }
+        }
+        @if variable-exists(main-nav-item-border-bottom-color) {
+            // Set the bottom border color of the links in the dashboard like 'Courses', 'Programs', 'Profile' etc.
+            .main .nav-tab a {
+                &.active {
+                    border-bottom-color: $main-nav-item-border-bottom-color !important;
+                    border-bottom-style: solid !important;
+                }
+            }
+        }
+        @if variable-exists(main-nav-item-hover-border-bottom-color) {
+            .main .nav-tab a {
+                &:not(.active):hover {
+                    border-bottom-color: $main-nav-item-hover-border-bottom-color !important;
+                    border-bottom-style: solid !important;
+                }
+            }
+        }
+        @if variable-exists(user-dropdown-color) {
+            // Customize the color of the links in the user dropdown menu
+            .secondary .dropdown-user-menu .dropdown-item a {
+                color: $user-dropdown-color !important;
+            }
+        }
+    }
+    .main .nav-tab a, .secondary .nav-item a, .secondary .nav-item .toggle-user-dropdown {
+        // Set the color of the dropdown arrow of the user dropdowm menu
+        color: $link-color !important;
+    }
+    .secondary {
+        @if variable-exists(customize-sign-in-btn) {
+            // Customize the sign-in button on the home page
+            a.sign-in-btn {
+                @include btn-styles($btn-sign-in-bg, $btn-sign-in-color, $btn-sign-in-border-color, $btn-sign-in-hover-bg, $btn-sign-in-hover-color, $btn-sign-in-hover-border-color);
+            }
+        }
+        @if variable-exists(customize-register-btn) {
+            // Customize the register button on the home page
+            a.register-btn {
+                @include btn-styles($btn-register-bg, $btn-register-color, $btn-register-border-color, $btn-register-hover-bg, $btn-register-hover-color, $btn-register-hover-border-color);
+            }
+        }
+    }
 
-  .nav-links .main .nav-tab a {
-      color: $link-color;
-  }
-  .nav-links .main .nav-tab a.active,
-  .nav-links .main .nav-tab a:hover {
-    border-bottom-color: $link-color;
-    border-bottom-style: solid;
-  }
+}
 
-  span {
-    color: $header-font-color; // Header text color
-  }
+.login-register {
+    .form-field .field-link {
+        // Customize the color of the links in the login/registration form.
+        color: $link-color;
+    }
+    .action-primary {
+        @if variable-exists(customize-logistration-action-btn) {
+            // Customize the sign-in/registration button on the login page
+            &.login-button, &.register-button {
+                @include btn-styles($btn-logistration-bg, $btn-logistration-color, $btn-logistration-border-color, $btn-logistration-hover-bg, $btn-logistration-hover-color, $btn-logistration-hover-border-color);
+                box-shadow: none;
```

Comment:
@giovannicimolin, I had removed the box shadow because the default is a blue shadow which looks ugly when a different color palette is chosen for the buttons. And since the buttons look nice without the box shadow, I say we leave this as-is.

Reply:
@lgp171188 Looks good! :+1: